README.md

```diff
@@ -22,10 +22,31 @@
   subdirectory.
 
 
+## Configuring tests
+
+You can place an optional `config.yml` file in the recipe directory, next to `CMakeLists.txt`.
+If present, the test script will parse it to set environment variables and CMake definitions for a particular recipe.
+
+Example:
+```yaml
+# environment variables to be set
+env:
```

Comment:
I think the best practice for compilers is to pass `CMAKE_<LANG>_COMPILER` with `-D` rather than set environment variables.

Reply:
OK - I will update the example.